### PR TITLE
Release notes for Flow Designer 1.5.5

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 1]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4593
+You can now prevent connectors that are listed in Anypoint Exchange from being used in Mule applications. If you have permission to edit assets in Anypoint Exchange, you can add the `nofdsupport` tag to the list of tags for a connector; afterward, that connector will not be available for Mule applications being created in Design Center.
+
+=== Enhancements
+// PROJECTX-4838
+* When you drag and drop message objects from the Visual Function Editor into function fields, you can position the object anywhere in an existing expression.
+// PROJECTX-4041
+* In Transform cards, the Output Payload section is now expanded by default if the payload contains data types.
+// PROJECTX-3721
+* As in previous releases, when you are adding a card to a flow you can start typing the name in the *Select a Component* dialog to filter the list of components. Now, when you have filtered the list down to one component, you no longer have to click on that component to select it. You can simply type *Enter*.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: Optional: "a flow you can" ->  "a flow, you can"

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 2]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+
+=== Fixes
+
+//PROJECTX-4432
+* If changes were made to a Mule app while the status was “Starting Application”, the status continued to change to “Running” and the changes were not applied. The status should have reverted to “Ready to Run” and the changes applied.
+// PROJECTX-4441
+* The error count displayed on the sidebar could be different from the error counts displayed on a flow and at the top of the canvas.
+// PROJECTX-4464
+* On Microsoft Windows 10 and in the Edge browser, icons and names were too distant from each other in the display of the list of triggers in the Let’s Get Started wizard.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: Bold the UI component name: *Let's Get Started* wizard.

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 3]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4526
+* An error was not added to the error count if a driver was not specified in a Database card.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *Database* card

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 4]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4528
+* Cards for scopes, such as the For Each and Choice cards, require at least one card to follow them. If a scope card did not yet have a card following it, an error was counted both on the scope card and on the plus icon next to it.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *For Each* and *Choice* cards.

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 5]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4888
+* The Search endpoint that is exposed in the Mozart xAPI was filtering out assets from Anypoint Exchange that did not have mule-plugin files. However, this behavior became incorrect when REST APIs started to be converted to connectors automatically in Exchange. These connectors do not have mule-plugin files.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *Search* endpoint 
And should it be `mule-plugin` files, in a courier-like font?

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 6]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4902
+* In a Logger card, dragging the Type attribute into the Message field would remove any other attributes that were already in that field.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *Logger* card. I'm not sure about "Type attribute", but probably bold *Message* field.

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 7]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4903
+* In an HTTP Request card, if you typed the beginning of a URL in the URL field, dragged Custom Expression into the field from the Visual Function Editor on the right, created a custom expression, closed the card, and then reopened the card, the URL field would be empty.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *HTTP Request* card and *URL* field, *Custom Expression*

Comment (dukesphere) [reply modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: and *URL* field

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 8]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4940
+* In some cases, it was possible to click the *Upload* button to select a database driver to upload before the Database card populated a list of available drivers. If you tried to upload a driver that was already available to Design Center, then the error message "An asset already exists with that name and version" would appear and the upload would fail.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: *Database* card

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 9]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4941
+* Several problems that prevented log content from being copied easily have been fixed.
+// PROJECTX-4948
+* In some cases, it was difficult to delete a message object that had been dragged into a field from the Visual Function Editor.
+// PROJECTX-5122
+* The icon on a card would disapper if you downgraded to an earlier version of the asset that the card represented.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: "disapper" -> "disappear"

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 10]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-5123
+* Some cards were not sized correctly when they were opened.
+// PROJECTX-5132
+* The $ symbol was causing errors in the Visual Function Editor.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: $ -> `$`

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 11]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-5171
+* Dragging a message object from the Visual Function Editor into a field would sometimes add extra spaces to the end of the text input in that field. The spaces could be difficult to remove.
+// PROJECTX-5173
+* Adding an if/then/else statement to a message object that was in a field would prevent text input after the object.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: if/then/else -> `if`/`then`/`else`

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 12]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-5190
+* Dragging a message object into a field could delete white space that was in the field.
+// PROJECTX-5241
+* In some cases, mapping the output of a Transform card to a variable and then clicking the *Run* button could lead to an application being stuck in the *Starting Application* state.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *Transform* card

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 13]
```diff
@@ -26,6 +27,66 @@ These release notes cover the following versions of these features:
...
+// PROJECTX-4313
+* In the *Output Payload* pane of a Transform card, if you right-clicked an attribute, selected *Set Expression*, set the value, and then quickly clicked OK, the new value would not take effect. You had to set the value, pause for a few seconds, and then click *OK*.
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: bold: *Transform* card 
bold: *OK*

File: modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc [hunk 14]
```diff
@@ -752,4 +813,4 @@ This release of Design Center enables you to:
 
 * Users cannot move files to a folder via drag and drop in Firefox
 
-* API Designer does not delete baseUri parameter if header of file is changed to a fragment and mocking server is running
\ No newline at end of file
+* API Designer does not delete baseUri parameter if header of file is changed to a fragment and mocking server is running
```
Comment (dukesphere) [inline modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: baseUri -> `baseUri` 
"and mocking server is running" -> "and the mocking server is running."

Comment (rddheath) [reply modules/ROOT/pages/design-center/design-center-release-notes-mule-apps.adoc]: I'm going to skip this one. This line is for release notes that went out on July 29, 2017. The reason it shows up in the file diff is that there was a space change for some random reason.

**Pull request merged**